docs/config.md

```diff
@@ -78,6 +78,8 @@ Supported keys include:
 | `storage.docdb.url` | The go-cloud URI reference to a docstore collection | `firestore://projects/[PROJECT]/databases/(default)/documents/[COLLECTION]?name_field=name`| |
 |`storage.grafeas.projectid`|The project ID to store occurrences|||
 |`storage.grafeas.noteid` (optional)|The note ID to link occurrences. If noteid is not provided, a name in the format of `tekton-<NAMESPACE>` will be used.|||
+|`storage.grafeas.notehint` (optional)|The note hint is a field in the ATTESTATION note. If it is not provided, the default `This attestation note was generated by Tekton Chains` will be used.|||
+|`storage.grafeas.builderversion` (optional)|The builder version is a field in the BUILD note. If it is not provided, `tekton.dev/v1beta1` will be used.|||
```

Comment:
This probably needs a better description i.e.

- we should probably distinguish between Grafeas attestations/builders vs SLSA/intoto attestations/builders.
- what other values could this field be set to? any consequences that users should look out for?

Reply:
> * we should probably distinguish between Grafeas attestations/builders vs SLSA/intoto attestations/builders.

AFAIK, the builder referred in grafeas notes is the same as the builder referred in the intoto statement. Grafeas note needs the `builderVersion` information, whereas builder.type and builder.id fields are needed in the intoto statement.

> * what other values could this field be set to? any consequences that users should look out for?

No. For the notehint and builderVersion, there are no rules on those string values.

Reply:
> AFAIK, the builder referred in grafeas notes is the same as the builder referred in the intoto statement. Grafeas note needs the `builderVersion` information, whereas builder.type and builder.id fields are needed in the intoto statement.

👍 I think where I got tripped up was I just assumed this was the intoto attestation, then was confused when I couldn't find an builder version field. Being explicit about what types this affects will help short-circuit the search.

> No. For the notehint and builderVersion, there are no rules on those string values.

https://github.com/grafeas/grafeas/blob/cd23d4dc1bef740d6d6d90d5007db5c9a2431c41/proto/v1/build.proto#L31 says

```
// Required. Immutable. Version of the builder which produced this build.
```

Should we be generating this from the Pipeline/Chains version rather than letting this be configurable via configmap?